views/angular_partials/car_select.slim

```diff
@@ -1,31 +1,37 @@
-.text-center
-  h1 I want to save money on my...
-  br
+.text-center.container
+  
+  .row
+    h1 I want to save money on my...
+    br
 
-  select.btn.btn-primary.btn-lg ng-model='selectedYear' ng-change='updateMakes()'
-    option disabled='' year
-    option ng-repeat='year in years'
-      | {{year}}
-  br
-  br
+    div.text-center ng-hide='waitIcon'
+      i.fa.fa-spinner.fa-spin.fa-5x
```

Comment:
Putting this inline with the Makes select box would look nicer. Try adding another `.col-md-1` and include this stuff. Will have to do some manipulations with the margins to get it work.